.storybook/preview-head.html

```diff
@@ -4,3 +4,16 @@
   href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap"
   rel="stylesheet"
 />
+<!-- Import EDD template js -->
+<script type="text/javascript" src="/js/libs/jquery.js"></script>
+<script type="text/javascript" src="/js/cagov.core.js"></script>
+
+<!-- Override default storybook style to make it compatible with EDD template -->
+<style>
+  .sb-show-main.sb-main-centered {
+    display: block !important;
+  }
+  .sb-show-main.sb-main-centered #root {
+    padding: 0 !important;
+  }
+</style>
```

Comment:
@lomky These are the css overrides I made to make the rendering work better. These are applied to the storybook wrapper divs. Let me know if you think there are any issues with these.

Reply:
my only concern is if we think this could disguise changes to the application CSS, and I think its very clearly scoped to storybook-only. so good by me